recipes/fastjet-cxx/meta.yaml

```diff
@@ -0,0 +1,139 @@
+{% set name = "fastjet-cxx" %}
+{% set version = "3.4.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://gitlab.com/fastjet/fastjet/-/archive/fastjet-{{ version }}/fastjet-fastjet-{{ version }}.tar.gz
+    sha256: 8ce8b9be5cd0fba4f2fe914871eabe259dfa7524edf925635e20072cde2ae6a8
+  # manually include the siscone submodule for the plugin build
+  - folder: plugins/SISCone/siscone
+    git_url: https://gitlab.com/fastjet/siscone.git
+    git_rev: 19532d56e0d2f5da63047674254671827276c5ce
+
+build:
+  skip: true  # [win]
+  number: 0
+  missing_dso_whitelist:
+    - "$RPATH/libsiscone.so.0"  # [linux]
+    - "$RPATH/libsiscone_spherical.so.0"  # [linux]
+    - "lib/libsiscone.0.dylib"  # [osx]
+    - "lib/libsiscone_spherical.0.dylib"  # [osx]
```

Comment:
This is failing as during the link fixing it is looking for the deleted library

```
Fixing linking of /Users/runner/bld/fastjet-cxx_1733938101018/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/libfastjet.0.dylib in /Users/runner/bld/fastjet-cxx_1733938101018/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/libfastjetplugins.0.dylib
New link location is lib/libfastjet.0.dylib
Error: Could not find lib/libsiscone_spherical.0.dylib
```

Reply:
Can try an `post-link.sh`. https://docs.conda.io/projects/conda-build/en/latest/resources/link-scripts.html

Reply:
Maybe `no_link`? https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#no-link

Reply:
@conda-forge/help-c-cpp do you have suggestions on what to do in this case? It is unclear to me how to try to affect things after the build but before the link fixing.